github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-presubmits.yaml

```diff
@@ -93,3 +93,35 @@ presubmits:
               - "/bin/sh"
               - "-c"
               - "automation/check-patch.e2e-okd.sh"
+    - name: pull-kubernetes-nmstate-e2e-ocp
+      skip_branches:
+        - release-\d+\.\d+
+      annotations:
+        fork-per-release: "true"
+      always_run: false
+      optional: true
+      decorate: true
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      max_concurrency: 6
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror: "true"
+        preset-shared-images: "true"
+        preset-kubevirtci-quay-credential: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "yum install -y jq && cat $QUAY_PASSWORD | docker login --username $(cat $QUAY_USER) --password-stdin=true quay.io && automation/check-patch.e2e-ocp.sh"
```

Comment:
Would be nicer if we instdall jq at the provider not here :-/, so we don't have to do yum install at every lane.

Reply:
btw maybe we dont even need to install
on the ocp-4.3 lane of kubevirt we dont install, lets not install and see what happens ?
it would fail directly at the begining of cluster-up since it wont have jq to parse the credits
so we will see it

Reply:
Per Or's suggestion, I'm dropping the install for now.

Reply:
I am using jq from a container here https://github.com/kubevirt/kubevirtci/pull/231 so it does no tdepend on underlaying system.